extensions/dockerfiles/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,10 @@
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "Dokcerfiles"
+metadata:
+  keywords:
+  - "dockerfiles"
+  guide: "https://quarkus.io/guides/dockerfiles"
```

Comment:
Please only reference a guide here when you have one or @holly-cummins will be unhappy :).